### PR TITLE
mount: add example of mounting for a specific user

### PR DESCRIPTION
**Pull request by CyrusYip**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by CyrusYip** [comment]
A non root user will has write permission if mounting in this way.

File: pages/common/mount.md [hunk 1]
```diff
@@ -15,6 +15,10 @@
 
 `mount --mkdir {{path/to/device_file}} {{path/to/target_directory}}`
 
+- Mount a device to a directory for a specific user:
+
+`mount -o uid=userID,gid=groupID {{path/to/device_file}} {{path/to/target_directory}}`
```
Comment (kbdharun) [inline pages/common/mount.md:20]:
```suggestion
`mount -o uid={{uid}},gid={{gid}} {{path/to/device_file}} {{path/to/target_directory}}`
```
or
```suggestion
`mount -o uid={{user_id}},gid={{group_id}} {{path/to/device_file}} {{path/to/target_directory}}`
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by EmilySeville7cfg — APPROVED** [review]
- No issues found, approved.
- This page should be reviewed by `mount` users.
- Thanks for a contribution! 🚀

**Review by mfrw — APPROVED** [review]
LGTM :)
Thank you @CyrusYip for your contribution

**Pull request merged**